app-modules/archeo/src/Http/Controllers/Api/ActivityController.php

```
@@ -143,6 +144,10 @@ public function show(string $activityNumber): ActivityResource
             ->with(['galleries.media', 'media'])
             ->firstOrFail();
 
+        $activity->getMedia('pdfs')
+            ->reject(fn ($pdf) => $pdf->hasGeneratedConversion('watermarked'))
+            ->each(fn ($pdf) => WatermarkPdfJob::dispatch($pdf->id));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Archeo\Jobs\WatermarkPdfJob' in method 'show'. (undefined)

(StaticAccess)